docs/content.zh/docs/deployment/resource-providers/standalone/docker.md

```diff
@@ -431,16 +431,26 @@ services:
   ```
   You can then start creating tables and queries those.
 
-* Note, that all required dependencies (e.g. for connectors) need to be available in the cluster as well as the client.
-  For example, if you would like to use the Kafka Connector create a custom image with the following Dockerfile
+* 请注意，集群和客户端中都需要安装各种运行库包括各种Source和Sink连接器。
+  例如，如果你想使用卡夫卡（SQL Kafka） 连接器，那么你需要构建一个自定义Docker镜像。
+
+  创建一个名为 `kafka.Dockerfile` 的 Dockerfile，内容如下：
```

Comment:
Since the Chinese page hasn't been translated, English will do for now.

Reply:
Sounds good. I have updated the code and used `vimdiff` to make sure the change is the same in both files.